Week-6/Problem_6.5/pairWithDifference.js

```diff
@@ -0,0 +1,25 @@
+function pairWithDifference(arr, element) {
+  let length = arr.length;
+  for (let i = 0; i <= length; i++) {
+    for (let j = 0; j <= length; j++) {
+      if (i !== j) {
+        if (arr[i] - arr[j] === element) {
+          return 1;
+        }
+      }
+    }
+  }
+  return 0;
+}
```

Comment:
Time complexity is O(n^2) , using a hash map this can be achieved in linear time with extra space.